common/lib/xmodule/xmodule/tests/test_tabs.py

```diff
@@ -735,6 +744,7 @@ def test_tabs_with_discussion(self):
             tab_list=self.tabs_with_discussion,
             expected_discussion_link="default_discussion_link",
             expected_can_display_value=True,
+            is_hideable=True,
```

Comment:
@mtyaka That's what I did first, but I moved it to reduce code duplication.  I think you are right, though, it's kind of a weird place for the test, and it's only a few lines of code that get duplicated.